### DISCUSSION TITLE
在封装过程中，证明miner内存不停地再增大

### DISCUSSION THREAD

**Discussion by Not-is-True** [filecoin-project/lotus#8312] [General Q&A]
![image](https://user-images.githubusercontent.com/88710575/158128340-df3bee83-6fb5-46f5-b8ea-d9de1548b91a.png)
我们之前的miner的内存为128G，后来加到了512G后下发任务以及证明在同一个Miner。直接导致miner自动重启，有没有人遇到这个错误

**Comment by TippyFlitsUK** [comment]
Hi @Not-is-True 

Memory issues like this in Lotus are often caused by pledging too many sectors at the same time. 

Are you still experiencing OOM issues?

Many Thanks!
